v3/integrations/nrmongo-v2/example/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/newrelic/go-agent/v3/integrations/nrmongo-v2"
+	newrelic "github.com/newrelic/go-agent/v3/newrelic"
+	"go.mongodb.org/mongo-driver/v2/bson"
+	"go.mongodb.org/mongo-driver/v2/mongo"
+	"go.mongodb.org/mongo-driver/v2/mongo/options"
+)
+
+func main() {
+	app, err := newrelic.NewApplication(
+		newrelic.ConfigAppName("Basic Mongo App"),
+		newrelic.ConfigLicense(os.Getenv("NEW_RELIC_LICENSE_KEY")),
+		newrelic.ConfigDebugLogger(os.Stdout),
+	)
+	if nil != err {
```

Comment:
Avoid yoda conditionals
```suggestion
	if err != nil {
```